R/util-ParseThreshold.R

```diff
@@ -15,14 +15,16 @@
 
 ParseThreshold <- function(strThreshold) {
   # Parse from a comma separated string to a vector of numeric values
-  vThreshold <- strsplit(strThreshold, ",")[[1]] %>% as.numeric()
```

Comment:
![image](https://github.com/user-attachments/assets/8ccf120d-194c-4ac7-81f1-f4b64bd4da8e)

The `as.numeric()` portion of this would've already triggered a warning if the coercion couldn't be done, I'm assuming with the cli messages below the intention was to overwrite it?